server/config/config.go

```diff
@@ -34,15 +38,39 @@ type BaseConfig struct {
 
 	// InterBlockCache enables inter-block caching.
 	InterBlockCache bool `mapstructure:"inter-block-cache"`
+}
 
-	Pruning              string `mapstructure:"pruning"`
-	PruningKeepEvery     string `mapstructure:"pruning-keep-every"`
-	PruningSnapshotEvery string `mapstructure:"pruning-snapshot-every"`
+// ListenerConfig defines the API listener configuration.
+type ListenerConfig struct {
+	// Enable defines if the API server should be enabled.
+	Enable bool `mapstructure:"enable"`
+
+	// EnableUnsafeCORS defines if CORS should be enabled (unsafe - use it at your own risk)
+	EnableUnsafeCORS bool `mapstructure:"enabled-unsafe-cors"`
+
+	// Address defines the API server to listen on
+	Address string `mapstructure:"address"`
+
+	// MaxOpenConnections defines the number of maximum open connections
+	MaxOpenConnections uint `mapstructure:"max-open-connections"`
+
+	// RPCReadTimeout defines the Tendermint RPC read timeout (in seconds)
+	RPCReadTimeout uint `mapstructure:"rpc-read-timeout"`
+
+	// RPCWriteTimeout defines the Tendermint RPC write timeout (in seconds)
+	RPCWriteTimeout uint `mapstructure:"rpc-write-timeout"`
+
+	// RPCMaxBodyBytes defines the Tendermint maximum response body (in bytes)
+	RPCMaxBodyBytes uint `mapstructure:"rpc-max-body-bytes"`
+
+	// TODO: TLS/Proxy configuration.
```

Comment:
is there an issue to track this TODO?

Reply:
Yes, https://github.com/cosmos/cosmos-sdk/issues/6420. I'll update.